osf/utils/social.py

```diff
@@ -0,0 +1,38 @@
+from django.apps import apps
+
+SOCIAL_NETWORKS = [
+    {
+        '_id': 'twitter',
+        'name': 'Twitter',
+        'base_url': 'https://twitter.com/'
+    },
+    {
+        '_id': 'facebook',
+        'name': 'Facebook',
+        'base_url': 'https://www.facebook.com/'
+    },
+    {
+        '_id': 'instagram',
+        'name': 'Instagram',
+        'base_url': 'https://instagram.com/'
+    },
+    {
+        '_id': 'linkedin',
+        'name': 'LinkedIn',
+        'base_url': 'https://www.linkedin.com/'
+    },
+    {
+        '_id': 'youtube',
+        'name': 'YouTube',
+        'base_url': 'https://www.youtube.com/'
+    }
+]
+
+def ensure_social_networks():
```

Comment:
I would do this as a data migration, so it only runs when it needs to. We have a bad habit of "ensure"ing things which means we read a json file full of licenses and shove them into the database every time the application starts.